src/js/themes/base.js

```diff
@@ -50,7 +48,7 @@ const lightColors = [
 const focusColor = accentColors[0];
 
 const colors = {
-  active: rgba(221, 221, 221, 0.5),
+  active: 'rgba(221, 221, 221, 0.5)',
```

Comment:
Same as above, did the previous rgba without string around still resolve to a string value?